LambdatestEcom/UITestFixture.cs

```diff
@@ -18,41 +19,38 @@ public async Task Setup()
             {
                 Headless = ciEnv == "true",
                 ExecutablePath = @"C:\Users\avkan\AppData\Local\ms-playwright\chromium-1129\chrome-win\chrome.exe"
+
             });
 
+            // Ensure state directory and file exist
+            var stateDirectory = Path.GetDirectoryName(StateFilePath);
+            if (!Directory.Exists(stateDirectory))
+            {
+                Directory.CreateDirectory(stateDirectory);
+            }
+
+            if (!File.Exists(StateFilePath))
+            {
+                await File.WriteAllTextAsync(StateFilePath, "{}");
+            }
+
+            // Load state when creating context
             context = await browser.NewContextAsync(new BrowserNewContextOptions
             {
                 ViewportSize = new ViewportSize
                 {
                     Width = 1920,
                     Height = 1080
-                }
-            });
-
-            await context.Tracing.StartAsync(new()
-            {
-                Title = $"{TestContext.CurrentContext.Test.ClassName}.{TestContext.CurrentContext.Test.Name}",
-                Screenshots = true,
-                Snapshots = true,
-                Sources = true
+                },
+                StorageStatePath = StateFilePath
             });
 
             page = await context.NewPageAsync();
```

Comment:
after the page is loaded with context, you need to open some account page and check is user is logged in. If it is not, then you need to login via UI here and save the state (that code which you created as separate test should be here because it should be ran before each test, but not only once and only when someone decides to run that separate test)